lib/bullet/rack.rb

```diff
@@ -32,16 +38,10 @@ def call(env)
 
     # fix issue if response's body is a Proc
     def empty?(response)
-      # response may be ["Not Found"], ["Move Permanently"], etc.
-      if rails?
-        (response.is_a?(Array) && response.size <= 1) ||
-          !response.respond_to?(:body) ||
-          !response_body(response).respond_to?(:empty?) ||
-          response_body(response).empty?
-      else
-        body = response_body(response)
-        body.nil? || body.empty?
-      end
+      # response may be ["Not Found"], ["Move Permanently"], etc, but
+      # those should not happen if the status is 200
+      body = response_body(response)
```

Comment:
Hi,

The new **empty?** check fails for responses with no body and first.

`undefined method `first' for #<Rack::Deflater::GzipStream:0x00007f8c53508908>
Bullet::Rack#response_body
bullet (6.0.2) lib/bullet/rack.rb, line 85
Bullet::Rack#empty?
bullet (6.0.2) lib/bullet/rack.rb, line 42`

Works fine in **bullet 5.9.0**.